### ISSUE TITLE
leapseconds url outdated

### ISSUE THREAD

**Issue by marzeck** [pyrocko/pyrocko#337]
Hey guys,
while trying to read DATACUBE raw data in snuffler we've approached a crucial error message on several machines:
```
Traceback (most recent call last):
  File "/usr/local/bin/snuffler", line 11, in <module>
    load_entry_point('pyrocko==2018.11.6', 'gui_scripts', 'snuffler')()
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/apps/snuffler.py", line 10, in main
    snuffler.snuffler_from_commandline(args=args)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/gui/snuffler.py", line 331, in snuffler_from_commandline
    store_interval=options.store_interval)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/gui/snuffler.py", line 129, in snuffle
    win.get_view().load(**kwargs_load)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/gui/pile_viewer.py", line 1491, in load
    update_progress=update_progress)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/pile.py", line 1007, in load_files
    self.add_files(load)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/pile.py", line 1010, in add_files
    for file in files:
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/pile.py", line 436, in loader
    substitutions=substitutions, mtime=mtime)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/pile.py", line 746, in __init__
    self.load_headers(mtime=mtime)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/pile.py", line 762, in load_headers
    substitutions=self.substitutions):
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/io/__init__.py", line 66, in load
    filename, format=format, getdata=getdata, substitutions=substitutions))
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/io/__init__.py", line 162, in iload
    filename, load_data=load_data, **add_args.get(format, {})):
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/io/datacube.py", line 406, in iload
    [x[0] + util.gps_utc_offset(x[0]) for x in util.read_leap_seconds2()],
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/util.py", line 1732, in read_leap_seconds2
    return parse_leap_seconds_list(fn)
  File "/usr/local/lib/python2.7/dist-packages/pyrocko/util.py", line 1713, in parse_leap_seconds_list
    t = int(toks[0]) + t0
ValueError: invalid literal for int() with base 10: '<!DOCTYPE'
```

I've tracked down the issue to the path of the file with the leap seconds (and the respective url where it is coming from). On a computer with an older pyrocko version the config file (~/.pyrocko/config.pf) points to  a correct file (_leap-seconds.list_, created Oct. 29 2018). But now it's just the html output.
When searching for the url: [http://www.ietf.org/timezones/data/leap-seconds.list](http://www.ietf.org/timezones/data/leap-seconds.list) in different browsers, you will get redirected to the main page of the IETF and can't access the leap-seconds file. 

Hopefully you can find an alternative.
Martin

**Comment by marzeck** [comment]
In case someone needs a quick and dirty solution for the same problem: here is an old version of that leap-seconds.list file. just copy it into your directory: 

> ~/.pyrocko/leap-seconds.list


[leap-seconds-file.zip](https://github.com/pyrocko/pyrocko/files/2724695/leap-seconds-file.zip)

**Comment by emolch** [comment]
The URL now has to be accessed via SSL, otherwise the URL hasn't changed: https://www.ietf.org/timezones/data/leap-seconds.list

Problem now is that this incorrect url is in the Pyrocko config files everywhere, and some unlucky users have the redirect html page in their leap-seconds.list... I will add some checks to the code.

**Comment by emolch** [comment]
Should be fixed with a493d32b70bc11274015614b124970d8127fe1fb.

**Comment by marzeck** [comment]
Perfect, thank you @emolch 
Issue is closed
